docs/source/gbd2017_models/causes/smoking_related_mortality/index.rst

```diff
@@ -0,0 +1,439 @@
+     - 493
+     - 8
+     - 
+     - 
+     - 
+   * - ischaemic stroke
+     - 495
+     - 2
+     - 
+     - 
+     - 
+   * - haemorrhagic stroke (Intracerebral hemorrhage)
+     - 496
+     - 2
+     - 
+     - 
+     - 
+   * - subarachnoid haemorrhage
+     - 497
+     - 2
+     - 
+     - 
+     - 
+   * - atrial fibrillation and flutter
+     - 500
+     - 11
+     - 
+     - 
+     - Age group start after lung cancer
+   * - aortic aneurysm
+     - 501
+     - 8
+     - 
+     - 
+     - 
+   * - peripheral arterial disease
+     - 502
+     - 13
+     - 
+     - 
+     - Age group start after lung cancer
+   * - chronic obstructive pulmonary disease
+     - 509
+     - 4
+     - 
+     - 
+     - 
+   * - other chronic respiratory diseases
+     - 520
+     - 2
+     - 
+     - 
+     - 
+   * - asthma
+     - 515
+     - 5
+     - 
+     - 
+     - 
+   * - peptic ulcer disease
+     - 527
+     - 5
+     - 
+     - 
+     - 
+   * - gallbladder and biliary tract diseases
+     - 534
+     - 5
+     - 
+     - 
+     - 
+   * - Alzheimer disease and other dementias
+     - 543
+     - 13
+     - 
+     - 
+     - Age group start after lung cancer
+   * - Parkinson disease (protective)
+     - 544
+     - 9
+     - 
+     - 
+     - Age group start after lung cancer
+   * - multiple sclerosis
+     - 546
+     - 9
+     - 
+     - 
+     - Age group start after lung cancer
+   * - type‐II diabetes
+     - 976
+     - 8
+     - 
+     - 
+     - 
+   * - rheumatoid arthritis
+     - 627
+     - 6
+     - 
+     - 
+     - 
+   * - low back pain
+     - 630
+     - N/A
+     - YLD only
+     - 
+     - Exclude due to nonfatal
+   * - cataracts
+     - 671
+     - N/A
+     - YLD only
+     - 
+     - Exclude due to nonfatal
+   * - macular degeneration
+     - 672
+     - N/A
+     - YLD only
+     - 
+     - Exclude due to nonfatal
+   * - Fracture
+     - N/A
+     - N/A
+     - YLD only
+     - 
+     - Exclude due to nonfatal
+
+.. note::
+
+  The starting age group IDs were extracted according to the YLL start, not the YLD start.
+
+  The :ref:`Vivarium CSU Lung Cancer Screening  Model  <2017_concept_model_vivarium_swissre_lungcancer>` only models simulants 15 years (age group ID 8) and older, so notes were made for causes with age group starts older than 15 years only.
+
+Restrictions
+++++++++++++
+
+See the table above.
+
+Vivarium Modeling Strategy
+--------------------------
+
+Scope
++++++
+
+The Vivarium modeling strategy outlined in this document will be a **mortality only** model that utilizes forecasted cause-specific mortality rates for ischemic heart disease (IHD) and chronic obstructive pulmomary disease (COPD) from 2020 to 2040. These causes were selected because they represent a large share of all smoking-related mortality without needing to model several additional causes with smaller contributions. This model is intended to be pair with the :ref:`smoking risk exposure model <2017_risk_exposure_smoking_forecasted>`.
```

Comment:
I know we've gone back and forth on including stroke.  If you have decided for sure not to include, let's state that and include a justification.